public/js/pwas-view-transition.js

```diff
@@ -44,7 +44,11 @@ function updateColors() {
   const chartElement = document.getElementById('chart');
   new window.Loader(chartElement, 'dark-primary-background').show();
 
-  pwaDetails.classList.add('fadeIn');
+  // fade in cards
+  const cards = document.getElementsByClassName('detail-general');
```

Comment:
I get this lint errors:
/Users/julian/GitHub/gulliver/public/js/pwas-view-transition.js
  49:8   error  'card' is not defined        no-undef
  50:5   error  'card' is not defined        no-undef
  50:34  error  Trailing spaces not allowed  no-trailing-spaces